.vuepress/config.js

```diff
@@ -365,6 +365,7 @@ module.exports = {
           'b587b006a5e827e320046fb036a15f219a14c3f05f21224c1fc7a9c00f4504ca',
         indexUid: 'docs',
         placeholder: 'Search as you type...',
```

Comment:
Any reason why we don't use the ellipsis char?

```suggestion
        placeholder: 'Search as you type…',
```